### PR TITLE
Tutorial: Fix typo "of"->"or" 

### PR DESCRIPTION
**Pull request by Flap-Py**
Closes https://github.com/coala/coala/issues/2781

### CODE REVIEW

**Comment by sims1253** [comment]
ack 6676836

**Comment by sims1253** [comment]
unack 6676836

can you give the full link to the issue so github links it properly cross repository?

File: Users/Tutorial.rst [hunk 1]
```diff
@@ -351,7 +351,7 @@ To get help on using a bear or to get a description of the bear, use the
     coala --bears=SpaceConsistencyBear --show-bears --show-description
```
Comment (Adrianzatreanu) [inline Users/Tutorial.rst]: first of all, for the commit shortlog: It is not explicit enough. a better oen would be: ``Tutorial: Fix typo "of"->"or"``

as for the body, it is unneeded here at all, as the shortlog is explicit enough :) the body is optional, for situations like this

and for the last line, please give the full issue URL, as #2781 is not going to change anything in the correct repo :D

Comment (Flap-Py) [inline Users/Tutorial.rst]: I was trying to open the issue again to get the full url, but it opens this page only.
How can I get the full url now?

Comment (Adrianzatreanu) [inline Users/Tutorial.rst]: https://github.com/coala/coala/issues/2781 this is the full url.

Comment (Adrianzatreanu) [inline Users/Tutorial.rst]: i added an intentional space between ``/`` and the number, you should remove it and use that link

**Comment by Adrianzatreanu** [comment]
ack 3f38a1d5a2ffb05b54cb50970cb80a4ee2eee284

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Comment by Adrianzatreanu** [comment]
it looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

We also have a quick video tutorial on how to rebase: [https://asciinema.org/a/78683](https://asciinema.org/a/78683). That should help you understand the basics of how it works and what you should be doing.

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

(Be sure to use the coala-bears URL if you're on coala bears!)

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix it up for you - just tell us :).

Ping us on [our gitter channel](https://coala.io/chat) if you need help!

**Comment by Adrianzatreanu** [comment]
reack f3a8c0d

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**